data.json

```diff
@@ -1300,5 +1300,12 @@
     "githubUsername": "Mo-Shakib",
     "imageUrl": "https://user-images.githubusercontent.com/50780268/131604251-f07f6811-d795-448f-a7b8-0f13b58c602f.png",
     "issueNumber": 675
+  },
+
+  {
+    "name": "Komal Pal",
+    "githubUsername": "Komal7209",
+    "imageUrl": "https://user-images.githubusercontent.com/51391473/131920342-adaf32bd-cfde-4bb2-ad5f-390b61898a37.png",
```

Comment:
So, you're going to use this image? If not, let me know. 🙂

![image](https://user-images.githubusercontent.com/33154959/131854111-cff5bad4-fd84-46e8-abf0-3ce469e83fca.png)

Reply:
@Panquesito7 is this image not fine?

Reply:
IMO, the current image is too long, and it's hard to read and understand. It also looks blurry.